config/manager.go

```diff
@@ -42,6 +45,7 @@ func LoadConfig() {
 	}
 
 success:
+	log.Println(color.HiYellowString("Start loading file."))
```

Comment:
This is a little annoying.

Reply:
Without prompting this, the player has no way of knowing the status of the reload